WordPress/Jetpack/Classes/NUX/New Landing Screen/Views/JetpackLandingScreenView.swift

```diff
@@ -1,20 +1,68 @@
-import SwiftUI
+import UIKit
 
-struct JetpackLandingScreenView: View {
+final class JetpackLandingScreenView: UIView {
 
     private let prompts = JetpackPromptsConfiguration.Constants.basePrompts
     private let evenColor = JetpackPromptsConfiguration.Constants.evenColor
     private let oddColor = JetpackPromptsConfiguration.Constants.oddColor
+    private let lineSpacing: CGFloat = -6
+    private let interitemSpacing: CGFloat = -2
```

Comment:
Negative spacing has always felt like a code smell in these cases 😅 (as if something's too tall to begin with), but it looks like it works, so LGTM!

Reply:
I agree! Fonts have a `lineHeight` property ( it's 47.7 for system bold font ) and I think that's the one we should be tweaking, but it's read-only 😞.

Also, negative line spacing have no effect on `UILabel` but it does work on `UITextView`, which is weird! I'm not sure how Apple intended the `lineSpacing` to be used, but that seems the only way to reduce line spacing.

There is the `lineHeightMultiple` property but it doesn't seem to me the right property to use in this case. According to Apple:
>The natural line height of the receiver is multiplied by this factor (if positive) before being constrained by minimum and maximum line height. The default value of this property is 0.0.  

I think what we want is to add / subtract from the line height instead of multiplying it by a certain number.

Reply:
Hi @salimbraksa I agree with @twstokes that this kinda feels like a hack. If we think that this is a good improvement for our users, then by all means we can go on and merge this; on the other hand, I thought we had decided to not pursue this, and the issues themselves are marked as low priority, so... 🤷‍♂️ 
I agree that SwiftUI at this point feels a bit out of place here